src/utils/tokenFunctions.ts

```diff
@@ -0,0 +1,6 @@
+export const getToken = () => localStorage.getItem("accessToken");
+
+export const saveToken = (token: string) =>
+  localStorage.setItem("accessToken", token);
```

Comment:
This is a great read I can reccommend
[Best Practices for Storing Access Tokens in the Browser](https://thenewstack.io/best-practices-for-storing-access-tokens-in-the-browser/)